cms/static/sass/elements/_xblocks.scss

```diff
@@ -116,6 +116,14 @@
     }
   }
 
+  span {
```

Comment:
Nit: try not to target elements in general like this. If this isn't always going to be correct for ``.icon-move`` then find a parent class to make it more specific.